### ISSUE TITLE
Upload button hidden under profile button

### ISSUE THREAD

**Issue by Antonim54** [starburst997/notessimo-issues#44]
Upload button would be more accessible beside the profile button, as opposed to in the dropdown menu.

**Comment by starburst997** [comment]
True! I was thinking about adding it also on the "Browse" page on the top right

![image](https://user-images.githubusercontent.com/77518/109405540-97b42c00-793f-11eb-9b8e-bacd555c9956.png)

What do you think?

**Comment by Antonim54** [comment]
I was thinking you could put it here, so you could still click it regardless of what page you're on.
<img width="448" alt="Screen Shot 2021-02-27 at 9 48 35 PM" src="https://user-images.githubusercontent.com/70923723/109406320-08f6dd80-7946-11eb-97ea-6c85c49c70b5.png">
EDIT: Ah, misread that. Although a second button on the browse page in that position would read as a little redundant.

**Comment by starburst997** [comment]
Cool! I like that!
